Tensile/KernelWriter.py

```diff
@@ -5098,7 +5099,12 @@ def getSourceFileString(self, kernel):
         return (0, self.getKernelSource(kernel))
 
     except subprocess.CalledProcessError as exc:
-      print(exc)
+      if isinstance(exc.cmd, collections.Sequence):
```

Comment:
I am not familiar with error handling code.
1) Why we need to check the type of exc.cmd? Can it be a different type?
2) How the output message changed (before and after)?
3) It looks like exc also has output (or stdout, stderr). Don't we need to print it as well?

Reply:
subprocess accepts arguments as either a string or a sequence, most places in our code pass a list, but the check ensures both cases are treated correctly.
When the command is a list, the default stringification for the exception prints it to look like a python list like
Command ['cmd', '-arg1', 'val1', ...]
To make it easier to debug, this change reformats the list to the actual command-line string so you can easily copy-paste and re-run the failed command by itself.
Printing stdout+stderr is already taken care of elsewhere in the code.

Reply:
Thanks for your quick response.
I do not have any further comment.
Waiting for comments(or approval) from others.